packages/contracts/package.json

```diff
@@ -57,5 +57,9 @@
       "postinstall-postinstall": false,
       "husky": false
     }
+  },
+  "devDependencies": {
+    "@types/node": "^18.11.18",
+    "typescript": "^4.7.4"
```

Comment:
Couldn't this be a later version of typescript?

Reply:
the typescript version `^4.7.4` was suggested by some deps,
but it gets resolved into `4.9.4` (<5) at the end.
![image](https://user-images.githubusercontent.com/109317559/211735056-de942178-1d41-4d39-9a7e-fe67eb9f991c.png)
@pavlovcik

Reply:
So then shouldn't we upgrade it in the package to reflect the latest?

Reply:
> So then shouldn't we upgrade it in the package to reflect the latest?

bumped the typescript version.